### PR TITLE
Create CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by curquiza**
Related to https://github.com/meilisearch/integration-guides/issues/33

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+```bash
+$ bundle exec rspec spec/meilisearch/index/search_spec.rb -e 'does a basic search in index'
+```
+
+### Linter
+
+Each PR should pass the linter to be accepted.
+
+```bash
+# Check the linter error
```
Comment (eskombro) [inline CONTRIBUTING.md:65]:
```suggestion
# Check the linter errors
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+$ bundle exec rubocop lib/ spec/
+# Auto-correct
```
Comment (eskombro) [inline CONTRIBUTING.md:67]:
```suggestion
# Auto-correct the linter errors
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+$ bundle exec rubocop -a lib/ spec/
+```
+
+Once you think the remaining linter errors as valid, do not add any `rubocop` comment line in the code.<br>
```
Comment (eskombro) [inline CONTRIBUTING.md:71]:
```suggestion
If you think the remaining linter errors are acceptable, do not add any `rubocop` in-line comments in the code.<br>
```

**Comment by curquiza** [comment]
> Shouldn't we add a link in the README to the CONTRIBUTING?

Yes I think it's a good idea, can you suggest something (= what and where) in the [right issue](https://github.com/meilisearch/integration-guides/issues/33)? 🙂

**Comment by curquiza** [comment]
@eskombro I added your suggestion in the readme 🚀 !!

**Pull request merged**